### PR TITLE
Fix documentation link

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,9 +109,8 @@ All contents of this package are licensed under the [MIT license].
 [Composer Plugin]: https://github.com/puli/composer-plugin
 [Bernhard Schussek]: http://webmozarts.com
 [The Community Contributors]: https://github.com/puli/manager/graphs/contributors
-[Getting Started]: http://docs.puli.io/en/latest/getting-started.html
+[Installation guide]: hhttp://docs.puli.io/en/latest/installation.html
```
Comment (webmozart) [inline README.md]: Typo hhttp

Comment (webmozart) [inline README.md]: Could you fix this please?

Comment (tgalopin) [inline README.md]: I forgot, I do that now.

Comment (webmozart) [inline README.md]: Thanks :)

**Comment by tgalopin** [comment]
Fixed :) .

**Pull request merged**